src/analytics/analytics.ts

```diff
@@ -8,6 +8,10 @@ export enum NOTIFICATIONS_EVENT_NAMES {
   PROJECT_UNLISTED = 'Project unlisted',
   PROJECT_EDITED = 'Project edited',
   PROJECT_BADGE_REVOKED = 'Project badge revoked',
+  PROJECT_BADGE_REVOKE_REMINDER = 'Project badge revoke reminder',
+  PROJECT_BADGE_REVOKE_WARNING = 'Project badge revoke warning',
+  PROJECT_BADGE_REVOKE_LAST_WARNING = 'Project badge revoke last warning',
+  PROJECT_BADGE_UP_FOR_REVOKING = 'Project badge up for revoking',
```

Comment:
@CarlosQ96 @WhyldWanderer @divine-comedian
have we created the autopilot journeys for these events?